include/pthread.h

```diff
@@ -0,0 +1,376 @@
+#ifndef _PTHREAD_H
```

Comment:
Missing license header.  Also, nitpick: Include guard `#define` is usually in another style: `__PTHREAD_H__`.